tests/nest_tests/nest_loops_integration_test.py

```diff
@@ -18,20 +18,28 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+import numpy as np
 import os
+import pytest
 import unittest
-import numpy as np
 
 import nest
 
+from pynestml.codegeneration.nest_tools import NESTTools
 from pynestml.frontend.pynestml_frontend import generate_nest_target
 
 
+nest_version = NESTTools.detect_nest_version()
+
+
 class NestLoopsIntegrationTest(unittest.TestCase):
     """
     Tests the code generation and working of for and while loops from NESTML to NEST
     """
 
+    @pytest.mark.skipif(nest_version.startswith("v2"),
```

Comment:
The `while` and `for` loops should be supported for all NEST versions, correct?

Reply:
Probably, but see the comment above, not a test for core functionality.